sim/mcuboot-sys/build.rs

```diff
@@ -53,8 +53,7 @@ fn main() {
         conf.file("../../ext/tinycrypt/lib/source/ecc_dsa.c");
         conf.file("../../ext/tinycrypt/lib/source/ecc_platform_specific.c");
 
-        conf.file("mbedtls/library/bignum.c");
-        conf.file("mbedtls/library/asn1parse.c");
+        conf.file("../../ext/mbedtls/src/asn1parse.c");
```

Comment:
Changing this directory needs to be accompanied by a similar change near the end of the function, where it calls `walk_dir()`, otherwise changes in these files won't cause rebuilds.